app/views/OrderPage/index.tsx

```diff
@@ -0,0 +1,181 @@
+import React, { useCallback, useState } from 'react';
+import { Alert, Button, Container, NumberInput, TextOutput } from '@the-deep/deep-ui';
+import { BsCashStack } from 'react-icons/bs';
+import { gql, useMutation, useQuery } from '@apollo/client';
+import { useLocation } from 'react-router-dom';
+
+import styles from './styles.css';
+import {
+    BookByIdQuery,
+    BookByIdQueryVariables,
+    SingleOrderMutation,
+    SingleOrderMutationVariables,
+} from '#generated/types';
+
+const ORDER = gql`
+    mutation SingleOrder ($bookId: Int!, $qty: Int!) {
+        placeSingleOrder(data: {bookId: $bookId, quantity: $qty}) {
+            errors
+            ok
+        result {
+            orderCode
+            id
+            status
+            totalPrice
+            }
+        }
+    }
+`;
+
+const BOOK_DETAIL = gql`
+query BookById ($id: ID!) {
+    book(id: $id) {
+      id
+      title
+      price
+    }
+  }
+`;
+
+type Book = NonNullable<BookByIdQuery['book']>
+
+interface ItemProps {
+    book: Book;
+    quantity: number;
+    changeQuantity: (qty: number) => void;
+}
+
+function OrderItem(props: ItemProps) {
+    const { book, quantity, changeQuantity } = props;
+    const { id, title, price } = book;
+    const handleQuantityChange = (value: number | undefined,
+        name: string,
+        event: React.FormEvent<HTMLInputElement> | undefined) => {
+        if (value) changeQuantity(value);
+        else changeQuantity(1);
```

Comment:
Let's not skip curly braces.

Also, it's better to just use ternary operator here